src/pages/MemberProfileQrCode/index.jsx

```diff
@@ -53,7 +53,7 @@ export const MemberProfileQrCode = () => {
             <ProfileContent>
               <ProfileAvatar data={ `${memberProfile.firstName} ${memberProfile.lastName}` } />
               <ProfileName>{ memberProfile.firstName } { memberProfile.lastName }</ProfileName>
-              <ProfilePosition className={ style.position }>{ memberProfile.permissions }</ProfilePosition>
+              <ProfilePosition className={ style.position }>{ memberProfile.position }</ProfilePosition>
```

Comment:
I think we should hide `ProfilePosition` for a viewer if it isn't set (`null`)

Reply:
done